doc/dev/perf_counters.rst

```diff
@@ -200,3 +202,50 @@ The actual dump is similar to the schema, except that average values are grouped
    }
  }
 
+.. _Labeled Perf Counters:
+
+Labeled Perf Counters
```

Comment:
This should be dropped -- labeled perf counters appeared in reef.

Reply:
Roger that, @idryomov. I'll drop it now.

Reply:
... and it's dropped. "Labeled Perf Counters" has been removed from the Quincy docs.

Note to near-future Zac: Add a note to the Reef and Squid docs informing the reader that "Labeled Perf Counters" are new in Reef.